admin_manual/configuration/server/security/password-policy.rst

```diff
@@ -8,23 +8,34 @@ Password Policy
 ---------------
 
 .. figure:: ../../../images/configuration/server/security/password-policy-app.png
-   :alt: The (enterprise-only) password policy application
+   :alt: The Password Policy application
 
-ownCloud Enterprise users have the option of enabling `the Password Policy app`_. 
-This application enables administrators to define password requirements, including: *minimum characters*, *numbers*, *capital letters*, and more, for all kinds of password endpoints, such as for user passwords and public link sharing passwords. 
-Moreover the admin can enforce expiration dates for public link shares.
+From the 2.0.0 release of `the Password Policy app`_, all ownCloud users (both enterprise and community) have the option of installing and enabling the application.
+The Password Policy application enables administrators to define password requirements, including:
+
+- Specify a valid password requirements.
```

Comment:
Specify requirements for a valid password,
Specify what is required for a password to be valid.
or...?

Reply:
=> ownCloud admins, not users (users don't install apps)

Reply:
Gotcha. The unrequired 'a' has been removed in my update (to be pushed shortly).

Reply:
Please add a hint on this page that when password expiration policies are imposed on an existing instance, the system administrator needs to run the occ command once to put the policies in place. This is a very important point to know for administrators because if they don't take this into account, policies simply won't work. I would maybe even add the detailed description here and link to the occ command descriptions only for describing it's usage. Lot's of 'describes' in the last sentence xD